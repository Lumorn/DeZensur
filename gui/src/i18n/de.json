{
  "file": "Datei",
  "add_images": "Bilder hinzufügen…"
}
