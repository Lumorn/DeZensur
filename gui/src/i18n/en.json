{
  "file": "File",
  "add_images": "Add Imagesâ€¦"
}
