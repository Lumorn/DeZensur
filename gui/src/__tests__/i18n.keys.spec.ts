import de from '../i18n/de.json';
import en from '../i18n/en.json';

test('de.json und en.json besitzen identische Keys', () => {
  const keysDe = Object.keys(de).sort();
  const keysEn = Object.keys(en).sort();
  expect(keysDe).toEqual(keysEn);
  expect(keysDe).toMatchSnapshot();
});
