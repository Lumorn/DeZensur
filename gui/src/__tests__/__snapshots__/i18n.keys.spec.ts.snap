// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`de.json und en.json besitzen identische Keys 1`] = `
[
  "add_images",
  "file",
]
`;
